src/components/pokedex.tsx

```diff
@@ -41,30 +50,42 @@ export function Pokedex() {
   };
 
   // Verificar si el Pokémon actual es favorito
-  const isCurrentPokemonFavorite = selectedPokemon ? favoritePokemon.some(p => p.name === selectedPokemon.name) : false;
+  const isCurrentPokemonFavorite = selectedPokemon
+    ? favoritePokemon.some((p) => p.name === selectedPokemon.name)
+    : false;
 
   return selectedPokemon ? (
     <div className={c("pokedex", `pokedex-${theme}`)}>
       <div className="panel left-panel">
         <h2 className="favorite-title">Pokemon Team</h2>
         <div className="favorite-pokemon-container">
-          {favoritePokemon.map((pokemon, index) => (
-            <div key={pokemon.name} className="favorite-pokemon-wrapper">
-              <img
-                src={pokemon.imageSrc}
-                alt={pokemon.name}
-                className="favorite-pokemon-image"
-              />
-              {(index + 1) % 2 === 0 && <br />} {}
-            </div>
-          ))}
+          {Array(6)
+            .fill(null)
+            .map((_, index) => (
+              <div key={index} className="favorite-pokemon-wrapper">
+                {favoritePokemon[index] && (
+                  <img
+                    src={favoritePokemon[index].imageSrc}
+                    alt={favoritePokemon[index].name}
+                    className="favorite-pokemon-image"
+                  />
+                )}
+              </div>
+            ))}
```

Comment:
second part for the grid layout thing: an interesting tool in js is to create  a "6 empty-elements array" (`Array(6).fill(null).map(() => {})`m in this case, we have an array of 6 favorite pokemons and what we do is to "fill" the next position of that array